### ISSUE TITLE
Question about mean subtract

### ISSUE THREAD

**Issue by jamesclyeh** [ry/tensorflow-resnet#19]
How come cafe model uses the mean subtracted image, while tensorflow model uses the original image?
caffe_model = load_caffe(img_p, layers)
vs.
o = sess.run(i, {images: img[np.newaxis, :]})

**Comment by danielgordon10** [comment]
To me it looks like those commands are baked into the network loaded in forward. It does bgr to rgb, scaling, and mean subtraction.

**Comment by jamesclyeh** [comment]
Yes I guess that is true, but that may be because the network was trained when the mean subtraction was still there. If you train from scratch and save now, mean subtraction is probably not there.
